torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1981,6 +1981,14 @@ def print_results(grouped, indent, result_fn):
 
 TLS = threading.local()
 
+@contextmanager
+def _disable_specialize_zero_one(shape_env):
+    prev = shape_env.specialize_zero_one
+    try:
+        shape_env.specialize_zero_one = False
+        yield
+    finally:
+        shape_env.specialize_zero_one = prev
```

Comment:
nuh uh, I'm not sure why you need this but we definitely shouldn't do it this way

Reply:
Yeah, I think I find some issue with current PR, will turn into WIP and request for review again.